lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -142,19 +155,10 @@ def main():
                     dl1_container.width = width
                     dl1_container.length = length
                     dl1_container.r = np.sqrt(dl1_container.x ** 2 + dl1_container.y ** 2)
+                    dl1_container.log_intensity = np.log10(dl1_container.intensity)
 
-                else:
-                    # for consistency with r0_to_dl1.py:
-                    for key in dl1_container.keys():
-                        dl1_container[key] = \
-                            u.Quantity(0, dl1_container.fields[key].unit)
-
-                    dl1_container.width = u.Quantity(np.nan, u.m)
-                    dl1_container.length = u.Quantity(np.nan, u.m)
-                    dl1_container.wl  = u.Quantity(np.nan, u.m)
-
-            for p in parameters_to_update:
-                params[ii][p] = u.Quantity(dl1_container[p]).value
```

Comment:
Is this patch no longer needed? The output in case the same cleaning is used is identical to the one we would obtain directly from the r0_to_dl1 script?

Reply:
> Is this patch no longer needed? The output in case the same cleaning is used is identical to the one we would obtain directly from the r0_to_dl1 script?

Ok, I see the test is supposed to check exactly that, so if it works, no problem.